manifests/params.pp

```diff
@@ -110,7 +110,7 @@
       $rundir                     = '/var/run/puppetlabs'
       $ssldir                     = '/etc/puppetlabs/puppet/ssl'
       $vardir                     = '/opt/puppetlabs/puppet/cache'
-      $sharedir                   = '/opt/puppetlabs/puppet'
+      $sharedir                   = '/opt/puppetlabs/puppet/modules'
```

Comment:
I think this is incorrect and shouldn't be changed.

Reply:
Oops, sorry to many versions that I've tried were the cause